diff -up kasts-25.08.2/CMakeLists.txt.omv~ kasts-25.08.2/CMakeLists.txt
--- kasts-25.08.2/CMakeLists.txt.omv~	2025-10-29 19:14:40.657113267 +0100
+++ kasts-25.08.2/CMakeLists.txt	2025-10-29 19:14:52.248565254 +0100
@@ -54,7 +54,7 @@ ecm_set_disabled_deprecation_versions(
     KF 5.240.0
 )
 
-find_package(Qt6 ${QT_MIN_VERSION} REQUIRED NO_MODULE COMPONENTS Core Quick Test Gui QuickControls2 Sql Svg Xml Network)
+find_package(Qt6 ${QT_MIN_VERSION} REQUIRED NO_MODULE COMPONENTS Core Quick Test Gui QuickControls2 Sql Svg Widgets Xml Network)
 find_package(KF6 ${KF_MIN_VERSION} REQUIRED COMPONENTS I18n CoreAddons Kirigami Syndication Config ThreadWeaver ColorScheme IconThemes)
 
 find_package(KF6 ${KF_MIN_VERSION} QUIET COMPONENTS BreezeIcons)
